### ISSUE TITLE
System.IndexOutOfRangeException thrown when executing BackwardDeleteWord via a binding

### ISSUE THREAD

**Issue by qcom** [PowerShell/PSReadLine#732]
I'm new to both Powershell and the PSReadLine libary but I encountered a `System.IndexOutOfRangeException` while customizing my bindings. Please let me know if I can provide any other information. I searched open issues and was not able to find this particular bug.

Environment data
----------------
PS version: 5.1.16299.98
PSReadline version: 1.2
os: 10.0.16299.15 (WinBuild.160101.0800)
PS file version: 10.0.16299.15 (WinBuild.160101.0800)

Steps to reproduce
--------------------------------------
1. Enter text at prompt (e.g. `foo bar bing`)
2. Execute the `BackwardDeleteWord` function via a binding (in my case `Alt+Backspace`)

Exception report
--------------------------------------

```
PS C:\> foo bar bing
Oops, something went wrong.  Please report this bug with the details below.
Report on GitHub: https://github.com/lzybkr/PSReadLine/issues/new
-----------------------------------------------------------------------
Last 13 Keys:
 f o o Spacebar b a r Spacebar b i n g Alt+Backspace

Exception:
System.IndexOutOfRangeException: Index was outside the bounds of the array.
   at System.Text.StringBuilder.get_Chars(Int32 index)
   at Microsoft.PowerShell.PSConsoleReadLine.ViFindPreviousWordPoint(Int32 i, String wordDelimiters)
   at Microsoft.PowerShell.PSConsoleReadLine.BackwardDeleteWord(Nullable`1 key, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)
```

**Comment by lzybkr** [comment]
Duplicate of #536 and fixed in [2.0.0-beta2](https://www.powershellgallery.com/packages/PSReadLine/2.0.0-beta2).

**Comment by qcom** [comment]
Thank you @lzybkr! I should have searched closed issues as well.
